pkgs/desktops/gnome/misc/gitg/default.nix

```diff
@@ -30,22 +31,13 @@
 
 stdenv.mkDerivation rec {
```

Comment:
IIRC this will not be backportable because it does not build with older meson.

Reply:
I honestly don't do backports by default unless I have reasons for that :shrug: